src/components/Select/__stories__/SelectShowcase.tsx

```diff
@@ -435,6 +435,25 @@ export const SelectShowcase = (props: SelectProps) => {
                 <Select.Option value="val3" content="Value3" />
                 <Select.Option value="val4" content="Value4" />
             </ExampleItem>
+            <ExampleItem
```

Comment:
I think we do not need such example - you can set all needed properties to particular values to get this state

Reply:
Yes, I've made it to show component to designers
[fe90b25](https://github.com/gravity-ui/uikit/pull/1368/commits/fe90b25e526db8974f688c54952d29c404de4d1f)